changelog.md

```diff
@@ -4,10 +4,36 @@ All notable end-user facing changes are documented in this file.
 ### [vNext]
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
-- Invoker codegen: methods returning Task<object> do not need Box() calls #2221
-- CodeGen: Avoid wrapping IGrainMethodInvoker.Invoke body in try/catch #2220
 - Several major performance improvements
 
+### [v1.3.1]
+- Improvements
+  - Ability to specify interleaving per message type (was needed for Orleankka) #2246
+  - Invoker codegen: methods returning Task<object> do not need Box() calls #2221
+  - CodeGen: Avoid wrapping IGrainMethodInvoker.Invoke body in try/catch #2220
```

Comment:
These 2 are also listed under "Performance". It should be removed from 1 of the 2 buckets